pkg/manager/member/scaler_test.go

```diff
@@ -158,3 +163,188 @@ func newFakeGeneralScaler() (*generalScaler, cache.Indexer, *controller.FakePVCC
 	return &generalScaler{pvcLister: pvcInformer.Lister(), pvcControl: pvcControl},
 		pvcInformer.Informer().GetIndexer(), pvcControl
 }
+
+func TestScaleOne(t *testing.T) {
```

Comment:
Could we add a case for "scale tidb to 0"?

Reply:
fixed some corner cases and add more tests in c8eb4058964cd5b7b6088b6d5221c0b17c8929f2